diagnostic.md

```diff
@@ -40,31 +45,33 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+  Modify the attribute of Json response.
   ```
 
   ```rb
   class ProfileSerializer < ActiveModel::Serializer
+  attributes :id, :given_name, :surname, :favorites
   end
   ```
 
 1.  What would the command be to _scaffold_ out a **join table** for Favorites from
 the above `Movies` and `Profiles`.
 
   ```sh
-    # < Your Response Here >
+  bin/rake generate scaffold favorites movies:references profiles:references
   ```
 
 1.  What is `Dependent: Destroy` and where/why would we use it?
 
   ```md
-    # < Your Response Here >
+  Break down the relationship for one of the two main entities. We will use it in join table only.
```

Comment:
`dependent: :destroy` will cause the deletion of an object to also delete all associated objects.  For example, if a `User` model has `has_many :favorites, dependent: :destroy`, then deleting a particular user will also delete all rows in the `favorites` table where `user_id` refers to the `id` of the user being deleted.